safe_transaction_service/history/indexers/safe_events_indexer.py

```diff
@@ -318,8 +318,8 @@ def _process_decoded_element(
             internal_tx_decoded = None
         elif event_name == "ChangedMasterCopy":
             internal_tx_decoded.function_name = "changeMasterCopy"
-            internal_tx.arguments = {
-                "_masterCopy": args.get("singleton") or args.get("masterCopy")
+            internal_tx_decoded.arguments = {
+                "_masterCopy": args.get("masterCopy") or args.get("singleton")
```

Comment:
> ChangedMasterCopy event was supposed to generate an argument _masterCopy instead of masterCopy 

It is still masterCopy.

Reply:
Yes but previously was not stored as `internal_tx` does not have an `arguments` field